examples/fft_physical_z/fft_r2c_z.f90

```diff
@@ -122,8 +131,9 @@ program fft_r2c_z
       call decomp_2d_fft_3d(out, in_r)
       t4 = t4 + MPI_WTIME() - t3
 
+      ! normalisation - note 2DECOMP&FFT doesn't normalise
       !$acc kernels
-      in_r = in_r / real(nx, mytype) / real(ny, mytype) / real(nz, mytype)
+      in_r = in_r / (real(nx, mytype) * real(ny, mytype) * real(nz, mytype))
```

Comment:
From experience it's better to do the triple division as the product can overflow for large grids (although this may not happen with the `real` conversion)